### PR TITLE
oneskyapp-pull-android-translations 0.0.1

### PR DESCRIPTION
**Pull request by prcaen**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/oneskyapp-pull-android-translations/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    is_dont_change_value: true
+    is_expand: true
+    is_required: true
+    is_sensitive: false
```
Comment (koral--) [inline steps/oneskyapp-pull-android-translations/0.0.1/step.yml]: It should be sensitive.

File: steps/oneskyapp-pull-android-translations/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  - name: git
+  - name: wget
+  - name: jq
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    is_dont_change_value: true
```
Comment (koral--) [inline steps/oneskyapp-pull-android-translations/0.0.1/step.yml]: What is the reason of don't change value?

File: steps/oneskyapp-pull-android-translations/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    is_expand: true
+    is_required: true
+    is_sensitive: false
+    title: Your own API Secret key
+  secret_key: $ONESKYAPP_SECRET_KEY
+- opts:
+    is_dont_change_value: true
```
Comment (koral--) [inline steps/oneskyapp-pull-android-translations/0.0.1/step.yml]: What is the reason of don't change value?

**Comment by prcaen** [comment]
Hello @koral-- thanks for your feedbacks.
I have made a new PR.
Thanks.

**Pull request closed without merging**